tests/blackbox/data/namespace-scoped/kyma_eventing_wrong_event_type/deployment.yml

```diff
@@ -3,39 +3,6 @@ kind: Namespace
 metadata:
   name: kyma-eventing-wrong-event-type
 ---
-# Eventing Manager CR with NATS backend
```

Comment:
Now creating the Eventing with NATS CR in cluster setup, instead of doing it in a test case.